### PR TITLE
remove non-informative error message that is printed on every restart

### PR DESCRIPTION
**Pull request by deepthi**
We run alter statements to add a `db_name` column to local_metadata and shard_metadata tables. These alters are only expected to succeed the first time, and are expected to fail on every subsequent vttablet restart.
The `Expected error` message is annoying and not helpful, hence it has been removed.

Signed-off-by: deepthi <deepthi@planetscale.com>

### CODE REVIEW

File: go/vt/mysqlctl/metadata_tables.go [hunk 1]
```diff
@@ -88,34 +88,32 @@ func PopulateMetadataTables(mysqld MysqlDaemon, localMetadata map[string]string,
...
+			if merr, ok := err.(*mysql.SQLError); !ok || merr.Num != mysql.ERDupFieldName {
 				log.Errorf("Unexpected error executing %v: %v", sql, err)
 				return err
 			}
 		}
 	}
 	sql := fmt.Sprintf(sqlUpdateLocalMetadataTable, dbName)
 	if _, err := conn.ExecuteFetch(sql, 0, false); err != nil {
-		log.Errorf("unexpected error executing %v: %v, continuing. Please check the data in _vt.local_metadata and take corrective action", sql, err)
+		log.Errorf("Unexpected error executing %v: %v, continuing. Please check the data in _vt.local_metadata and take corrective action.", sql, err)
```
Comment (sougou) [inline go/vt/mysqlctl/metadata_tables.go]: I think we should just remove 'Unexpected' from these messages (including the one above).

**Comment by rafael** [comment]
I think this good to merge after @sougou  comment is addressed

File: go/vt/mysqlctl/metadata_tables.go (hunk 1 above)
Comment (deepthi) [reply go/vt/mysqlctl/metadata_tables.go]: Done. I squashed the commits since the changes are so trivial.

**Comment by deepthi** [comment]
Comments have been addressed and CI is passing. I can merge this if one of @sougou / @rafael approves.

**Pull request merged**